lib/skate/settings/user.ex

```diff
@@ -75,12 +76,20 @@ defmodule Skate.Settings.User do
   end
 
   def is_in_test_group(user_id, test_group_name) do
-    %{test_groups: user_test_groups} =
-      DbUser
-      |> Skate.Repo.get(user_id)
-      |> Skate.Repo.preload(:test_groups)
+    user_id
+    |> get_by_id!()
+    |> all_test_group_names()
+    |> Enum.member?(test_group_name)
+  end
+
+  def all_test_group_names(user) do
+    user = Skate.Repo.preload(user, :test_groups)
```

Comment:
I would eventually like for this to be just one query, but I wasn't convinced that that was necessary right now. Unless there's something about Ecto that I'm missing, this will run 2 queries, not like.. 20.